common/templates/options.php

```diff
@@ -79,6 +79,27 @@
                     <?php endif; ?>
                 </td>
             </tr>
+            <tr valign="top">
+                <th scope="row">
+                    <label><?php _e('Order status for invitations', 'webwinkelkeur'); ?></label>
+                </th>
+                <td>
+                    <fieldset>
+                        <div style="height: 150px; overflow: auto;">
+                            <?php foreach (wc_get_order_statuses() as $key => $label): ?>
```

Comment:
You need to make sure the plugin works when WooCommerce is not installed.  For example, by checking that the `wc_get_order_statuses` function exists.  Also please make sure that you test the plugin when WooCommerce is disabled.

Reply:
Correct it throws an error, also https://github.com/webwinkelkeur/wordpress/blob/1c7de23d5d7e7a5be7c2e9edfdbf85a5dc7429e4/common/templates/options.php#L77 never shows up
Replaced it with https://docs.woocommerce.com/document/query-whether-woocommerce-is-activated/

Reply:
Looks like it should have been checking `Admin::$woocommerce`.  `Admin::$woocommerce` can be removed now, and the hook that sets it as well, I believe. You can do that in a separate PR if you like.

Product review functionality should also be disabled if WooCommerce is not activated.